spec/jwt_signed_request_spec.rb

```diff
@@ -5,17 +5,17 @@
 RSpec.describe JWTSignedRequest do
   describe '.sign' do
     it 'calls the Sign class' do
-      arguments = double :arguments
+      arguments = { arg: true }
```

Comment:
Consider using a double to provide more certainty that it's what gets passed (same applies below). 
```suggestion
      arguments = { arg: double }
```